src/Features/ERDDAP/Platform/Observations/CurrentConditions/test_data.ts

```diff
@@ -0,0 +1,16 @@
+import { prefferedDataTypes, prefferedDataTypesList, windDataTypes } from "./index"
+
+import { platform } from "stories/platform"
+
+export const wind_datasets = platform.properties.readings.filter((reading) =>
+  windDataTypes.has(reading.data_type.standard_name)
+)
+
+export const filtered_datasets = platform.properties.readings.filter(
+  (reading) => prefferedDataTypes.has(reading.data_type.standard_name) && reading.depth < 2
+)
+filtered_datasets.sort(
```

Comment:
Codacy found an issue: [Identifier 'filtered_datasets' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6071156)